.github/bin/build_docs.sh

```diff
@@ -36,12 +36,16 @@ TAR_NAME=${REF_VERSION_NAME##*/}
 
 if [[ "$REF_VERSION_NAME" == *"-tm"* ]] ;then
     DIR_VERSION_NAME=$(echo "$TAR_NAME" | sed -r "s/v*(.*)\.tar\.gz/\1/g")
-    TAR_DIR_NAME=incubator-celeborn-$DOC_LINK_PATH
-    wget "https://github.com/apache/incubator-celeborn/releases/download/${REF_VERSION_NAME}"
+    if [[ "$REF_VERSION_NAME" == *"-incubating"* ]] ;then
+      TAR_DIR_NAME=incubator-celeborn-$DOC_LINK_PATH
```

Comment:
why should we handle `incubator-*` case now?

Reply:
Because the tar we released earlier had an incubator.